controllers/services/auth/auth_controller.go

```diff
@@ -80,17 +55,3 @@ func NewServiceReconciler(ctx context.Context, mgr ctrl.Manager) error {
 
 	return nil
 }
-
-func shouldWatchDashboardConfig(ctx context.Context, request *types.ReconciliationRequest) bool {
-	d := resources.GvkToUnstructured(gvk.Dashboard)
-	if err := request.Client.Get(ctx, client.ObjectKey{Name: componentApi.DashboardInstanceName}, d); err != nil {
-		return false
-	}
-
-	c := resources.GvkToUnstructured(gvk.CustomResourceDefinition)
-	if err := request.Client.Get(ctx, client.ObjectKey{Name: odhDashboardConfigCRDName}, c); err != nil {
-		return false
-	}
-
-	return true
-}
```

Comment:
Just noticed it now and curious. What was this used for? To watch config just in case CRD and Dashboard are present?

Reply:
Yep, it's unneccessary now since we check in the create function.